File: C1.UWP.FlexChart/VB/FlexChartMvvmDemo/Services/IOrderService.vb
Imports FlexChartMvvmDemo.Model
Imports System.Threading.Tasks
Imports System.Text
Imports System.Linq
Imports System.Collections.Generic
Imports System


Namespace Services
    Public Interface IOrderService
        Sub FetchOrders(list As IList(Of OrderModel))
    End Interface
End Namespace

File: C1.UWP.FlexChart/CS/FlexChartMvvmDemo/Services/IOrderService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using FlexChartMvvmDemo.Model;

namespace FlexChartMvvmDemo.Services
{
    public interface IOrderService
    {
        void FetchOrders(IList<OrderModel> list);
    }
}
